2024-08-07 17:42:31,679 INFO    StreamThr :5697 [internal.py:wandb_internal():86] W&B internal server running at pid: 5697, started at: 2024-08-07 17:42:31.679428
2024-08-07 17:42:31,680 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: status
2024-08-07 17:42:31,681 INFO    WriterThread:5697 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/run-2nqbyuy9.wandb
2024-08-07 17:42:31,681 DEBUG   SenderThread:5697 [sender.py:send():382] send: header
2024-08-07 17:42:31,683 DEBUG   SenderThread:5697 [sender.py:send():382] send: run
2024-08-07 17:42:31,853 INFO    SenderThread:5697 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files
2024-08-07 17:42:31,853 INFO    SenderThread:5697 [sender.py:_start_run_threads():1136] run started: 2nqbyuy9 with start time 1723023751.679514
2024-08-07 17:42:31,859 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: check_version
2024-08-07 17:42:31,860 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: check_version
2024-08-07 17:42:31,965 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: run_start
2024-08-07 17:42:31,969 DEBUG   HandlerThread:5697 [system_info.py:__init__():32] System info init
2024-08-07 17:42:31,969 DEBUG   HandlerThread:5697 [system_info.py:__init__():47] System info init done
2024-08-07 17:42:31,969 INFO    HandlerThread:5697 [system_monitor.py:start():194] Starting system monitor
2024-08-07 17:42:31,969 INFO    SystemMonitor:5697 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-07 17:42:31,969 INFO    HandlerThread:5697 [system_monitor.py:probe():214] Collecting system info
2024-08-07 17:42:31,970 INFO    SystemMonitor:5697 [interfaces.py:start():190] Started cpu monitoring
2024-08-07 17:42:31,970 INFO    SystemMonitor:5697 [interfaces.py:start():190] Started disk monitoring
2024-08-07 17:42:31,970 INFO    SystemMonitor:5697 [interfaces.py:start():190] Started gpu monitoring
2024-08-07 17:42:31,971 INFO    SystemMonitor:5697 [interfaces.py:start():190] Started memory monitoring
2024-08-07 17:42:31,971 INFO    SystemMonitor:5697 [interfaces.py:start():190] Started network monitoring
2024-08-07 17:42:31,980 DEBUG   HandlerThread:5697 [system_info.py:probe():196] Probing system
2024-08-07 17:42:31,981 DEBUG   HandlerThread:5697 [system_info.py:_probe_git():181] Probing git
2024-08-07 17:42:31,983 DEBUG   HandlerThread:5697 [system_info.py:_probe_git():189] Probing git done
2024-08-07 17:42:31,983 DEBUG   HandlerThread:5697 [system_info.py:probe():244] Probing system done
2024-08-07 17:42:31,983 DEBUG   HandlerThread:5697 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-44-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-07T09:42:31.980325', 'startedAt': '2024-08-07T09:42:31.675084', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/llama2-7b', '--use_lora', 'True', '--init_lora_weights', 'pissa_niter_4', '--target_modules', 'q_proj,v_proj,k_proj,o_proj,gate_proj,down_proj,up_proj', '--lora_rank', '32', '--lora_alpha', '32', '--lora_dropout', '0', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:100000]', '--output_dir', '/home/rwkv/JL/out_model/llama-test', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '4', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': '74dce7f004b5b16bfd24c8ca22e737aee6ff0769'}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 2923.9483571428573, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 5499.903, 'min': 800.0, 'max': 5600.0}, {'current': 5500.0, 'min': 800.0, 'max': 5600.0}, {'current': 5391.935, 'min': 800.0, 'max': 5600.0}, {'current': 5500.0, 'min': 800.0, 'max': 5600.0}, {'current': 5500.072, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5500.0, 'min': 800.0, 'max': 5500.0}, {'current': 1677.964, 'min': 800.0, 'max': 5500.0}, {'current': 4300.391, 'min': 800.0, 'max': 4300.0}, {'current': 4300.289, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 474.66293716430664}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.55397415161133}}
2024-08-07 17:42:31,983 INFO    HandlerThread:5697 [system_monitor.py:probe():224] Finished collecting system info
2024-08-07 17:42:31,983 INFO    HandlerThread:5697 [system_monitor.py:probe():227] Publishing system info
2024-08-07 17:42:31,983 DEBUG   HandlerThread:5697 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-07 17:42:31,983 DEBUG   HandlerThread:5697 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-07 17:42:31,983 DEBUG   HandlerThread:5697 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-07 17:42:32,853 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/requirements.txt
2024-08-07 17:42:32,853 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/conda-environment.yaml
2024-08-07 17:42:33,778 DEBUG   HandlerThread:5697 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-07 17:42:33,778 DEBUG   HandlerThread:5697 [system_info.py:_save_code():90] Saving code
2024-08-07 17:42:33,780 DEBUG   HandlerThread:5697 [system_info.py:_save_code():111] Saving code done
2024-08-07 17:42:33,780 DEBUG   HandlerThread:5697 [system_info.py:_save_patches():128] Saving git patches
2024-08-07 17:42:33,801 DEBUG   HandlerThread:5697 [system_info.py:_save_patches():170] Saving git patches done
2024-08-07 17:42:33,802 INFO    HandlerThread:5697 [system_monitor.py:probe():229] Finished publishing system info
2024-08-07 17:42:33,803 DEBUG   SenderThread:5697 [sender.py:send():382] send: files
2024-08-07 17:42:33,803 INFO    SenderThread:5697 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-07 17:42:33,803 INFO    SenderThread:5697 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-07 17:42:33,804 INFO    SenderThread:5697 [sender.py:_save_file():1392] saving file diff.patch with policy now
2024-08-07 17:42:33,806 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: stop_status
2024-08-07 17:42:33,806 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: stop_status
2024-08-07 17:42:33,807 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-07 17:42:33,853 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/conda-environment.yaml
2024-08-07 17:42:33,853 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/diff.patch
2024-08-07 17:42:33,853 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/wandb-metadata.json
2024-08-07 17:42:33,853 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/code/llama/gmm.py
2024-08-07 17:42:33,853 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/code/llama
2024-08-07 17:42:33,853 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/code
2024-08-07 17:42:34,035 ERROR   wandb-upload_0:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=77b0a1f140500899451acb7cdcf60dd8507129e1c4f7617cadc0e6fc11a5d0e4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=77b0a1f140500899451acb7cdcf60dd8507129e1c4f7617cadc0e6fc11a5d0e4
2024-08-07 17:42:34,035 ERROR   wandb-upload_0:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5511', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:34,035 ERROR   wandb-upload_0:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2nqbyuy9/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E968F9AD4D8F73</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:34,046 DEBUG   SenderThread:5697 [sender.py:send():382] send: telemetry
2024-08-07 17:42:34,046 DEBUG   SenderThread:5697 [sender.py:send():382] send: config
2024-08-07 17:42:34,046 DEBUG   SenderThread:5697 [sender.py:send():382] send: metric
2024-08-07 17:42:34,046 DEBUG   SenderThread:5697 [sender.py:send():382] send: telemetry
2024-08-07 17:42:34,046 DEBUG   SenderThread:5697 [sender.py:send():382] send: metric
2024-08-07 17:42:34,046 WARNING SenderThread:5697 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-07 17:42:34,046 DEBUG   SenderThread:5697 [sender.py:send():382] send: telemetry
2024-08-07 17:42:34,046 DEBUG   SenderThread:5697 [sender.py:send():382] send: telemetry
2024-08-07 17:42:34,046 DEBUG   SenderThread:5697 [sender.py:send():382] send: config
2024-08-07 17:42:34,107 DEBUG   SenderThread:5697 [sender.py:send():382] send: exit
2024-08-07 17:42:34,107 INFO    SenderThread:5697 [sender.py:send_exit():589] handling exit code: 1
2024-08-07 17:42:34,108 INFO    SenderThread:5697 [sender.py:send_exit():591] handling runtime: 2
2024-08-07 17:42:34,110 INFO    SenderThread:5697 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 17:42:34,110 INFO    SenderThread:5697 [sender.py:send_exit():597] send defer
2024-08-07 17:42:34,110 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:34,111 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-07 17:42:34,111 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: defer
2024-08-07 17:42:34,111 INFO    SenderThread:5697 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-07 17:42:34,111 INFO    SenderThread:5697 [sender.py:transition_state():617] send defer: 1
2024-08-07 17:42:34,111 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:34,111 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-07 17:42:34,112 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: defer
2024-08-07 17:42:34,112 INFO    SenderThread:5697 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-07 17:42:34,112 INFO    SenderThread:5697 [sender.py:transition_state():617] send defer: 2
2024-08-07 17:42:34,112 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:34,112 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-07 17:42:34,112 INFO    HandlerThread:5697 [system_monitor.py:finish():203] Stopping system monitor
2024-08-07 17:42:34,112 DEBUG   SystemMonitor:5697 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-07 17:42:34,113 INFO    HandlerThread:5697 [interfaces.py:finish():202] Joined cpu monitor
2024-08-07 17:42:34,113 DEBUG   SystemMonitor:5697 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-07 17:42:34,113 INFO    HandlerThread:5697 [interfaces.py:finish():202] Joined disk monitor
2024-08-07 17:42:34,113 DEBUG   SystemMonitor:5697 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-07 17:42:34,157 INFO    HandlerThread:5697 [interfaces.py:finish():202] Joined gpu monitor
2024-08-07 17:42:34,157 INFO    HandlerThread:5697 [interfaces.py:finish():202] Joined memory monitor
2024-08-07 17:42:34,157 INFO    HandlerThread:5697 [interfaces.py:finish():202] Joined network monitor
2024-08-07 17:42:34,157 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: defer
2024-08-07 17:42:34,157 INFO    SenderThread:5697 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-07 17:42:34,157 INFO    SenderThread:5697 [sender.py:transition_state():617] send defer: 3
2024-08-07 17:42:34,157 DEBUG   SenderThread:5697 [sender.py:send():382] send: stats
2024-08-07 17:42:34,157 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:34,157 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-07 17:42:34,157 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: defer
2024-08-07 17:42:34,157 INFO    SenderThread:5697 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-07 17:42:34,157 INFO    SenderThread:5697 [sender.py:transition_state():617] send defer: 4
2024-08-07 17:42:34,157 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:34,157 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-07 17:42:34,157 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: defer
2024-08-07 17:42:34,157 INFO    SenderThread:5697 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-07 17:42:34,157 INFO    SenderThread:5697 [sender.py:transition_state():617] send defer: 5
2024-08-07 17:42:34,158 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:34,158 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-07 17:42:34,158 DEBUG   SenderThread:5697 [sender.py:send():382] send: summary
2024-08-07 17:42:34,158 INFO    SenderThread:5697 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 17:42:34,158 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: defer
2024-08-07 17:42:34,158 INFO    SenderThread:5697 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-07 17:42:34,158 INFO    SenderThread:5697 [sender.py:transition_state():617] send defer: 6
2024-08-07 17:42:34,158 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:34,158 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-07 17:42:34,158 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: defer
2024-08-07 17:42:34,158 INFO    SenderThread:5697 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-07 17:42:34,159 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: status_report
2024-08-07 17:42:34,216 ERROR   wandb-upload_1:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93b56afe6d26f7f3606969dc0019d62ba35af93e2a064b0e0bcd513d68456cba: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93b56afe6d26f7f3606969dc0019d62ba35af93e2a064b0e0bcd513d68456cba
2024-08-07 17:42:34,216 ERROR   wandb-upload_1:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:34,216 ERROR   wandb-upload_1:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2nqbyuy9/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E968F9B414B267</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:34,329 INFO    SenderThread:5697 [sender.py:transition_state():617] send defer: 7
2024-08-07 17:42:34,330 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:34,330 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-07 17:42:34,330 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: defer
2024-08-07 17:42:34,330 INFO    SenderThread:5697 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-07 17:42:34,589 ERROR   wandb-upload_2:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0857eef7e7867127aa5e226e226b37bcd5b929b263db0cf897db75c0c88f5b4f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0857eef7e7867127aa5e226e226b37bcd5b929b263db0cf897db75c0c88f5b4f
2024-08-07 17:42:34,590 ERROR   wandb-upload_2:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '284630', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:34,590 ERROR   wandb-upload_2:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2nqbyuy9/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2nqbyuy9/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E968F9B53C8751</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:34,854 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/config.yaml
2024-08-07 17:42:34,854 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/wandb-summary.json
2024-08-07 17:42:34,854 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/output.log
2024-08-07 17:42:35,107 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 17:42:35,453 ERROR   wandb-upload_0:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=77b0a1f140500899451acb7cdcf60dd8507129e1c4f7617cadc0e6fc11a5d0e4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=77b0a1f140500899451acb7cdcf60dd8507129e1c4f7617cadc0e6fc11a5d0e4
2024-08-07 17:42:35,453 ERROR   wandb-upload_0:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5511', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:35,453 ERROR   wandb-upload_0:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2nqbyuy9/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E968F9EE06A75F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:35,453 INFO    wandb-upload_0:5697 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=77b0a1f140500899451acb7cdcf60dd8507129e1c4f7617cadc0e6fc11a5d0e4

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 17:42:35,607 ERROR   wandb-upload_1:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93b56afe6d26f7f3606969dc0019d62ba35af93e2a064b0e0bcd513d68456cba: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93b56afe6d26f7f3606969dc0019d62ba35af93e2a064b0e0bcd513d68456cba
2024-08-07 17:42:35,607 ERROR   wandb-upload_1:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:35,607 ERROR   wandb-upload_1:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2nqbyuy9/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E968FA0600E8C3</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:35,608 INFO    wandb-upload_1:5697 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93b56afe6d26f7f3606969dc0019d62ba35af93e2a064b0e0bcd513d68456cba

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 17:42:36,066 INFO    SenderThread:5697 [sender.py:transition_state():617] send defer: 8
2024-08-07 17:42:36,066 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: poll_exit
2024-08-07 17:42:36,066 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:36,066 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-07 17:42:36,066 DEBUG   SenderThread:5697 [sender.py:send_request():409] send_request: defer
2024-08-07 17:42:36,066 INFO    SenderThread:5697 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-07 17:42:36,066 INFO    SenderThread:5697 [job_builder.py:build():298] Attempting to build job artifact
2024-08-07 17:42:36,067 INFO    SenderThread:5697 [job_builder.py:_get_source_type():428] is repo sourced job
2024-08-07 17:42:36,091 INFO    SenderThread:5697 [job_builder.py:build():404] adding wandb-job metadata file
2024-08-07 17:42:36,096 INFO    SenderThread:5697 [sender.py:transition_state():617] send defer: 9
2024-08-07 17:42:36,096 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:42:36,096 DEBUG   SenderThread:5697 [sender.py:send():382] send: artifact
2024-08-07 17:42:36,096 INFO    HandlerThread:5697 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-07 17:42:36,107 DEBUG   HandlerThread:5697 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 17:42:36,282 ERROR   wandb-upload_2:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0857eef7e7867127aa5e226e226b37bcd5b929b263db0cf897db75c0c88f5b4f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0857eef7e7867127aa5e226e226b37bcd5b929b263db0cf897db75c0c88f5b4f
2024-08-07 17:42:36,282 ERROR   wandb-upload_2:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '284630', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:36,282 ERROR   wandb-upload_2:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2nqbyuy9/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2nqbyuy9/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E968FA15F22D76</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:36,796 ERROR   wandb-upload_4:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/502/3ec9d0b0f39e55d0938d525d881bb305?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094236Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b6781eb18bf71ed70b954bedcaf941405d1d7f35bbf9ab8a829dfe61c0f3c284: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/502/3ec9d0b0f39e55d0938d525d881bb305?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094236Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b6781eb18bf71ed70b954bedcaf941405d1d7f35bbf9ab8a829dfe61c0f3c284
2024-08-07 17:42:36,797 ERROR   wandb-upload_4:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'PsnQsPOeVdCTjVJdiBuzBQ==', 'Content-MD5': 'PsnQsPOeVdCTjVJdiBuzBQ==', 'Content-Type': 'application/json', 'Content-Length': '31083', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:36,797 ERROR   wandb-upload_4:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/502/3ec9d0b0f39e55d0938d525d881bb305</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/502/3ec9d0b0f39e55d0938d525d881bb305</Resource><Region>wandb-local</Region><RequestId>17E968FA4CEEA10B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:36,855 INFO    Thread-12 :5697 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_174231-2nqbyuy9/files/output.log
2024-08-07 17:42:37,706 ERROR   wandb-upload_0:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=77b0a1f140500899451acb7cdcf60dd8507129e1c4f7617cadc0e6fc11a5d0e4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=77b0a1f140500899451acb7cdcf60dd8507129e1c4f7617cadc0e6fc11a5d0e4
2024-08-07 17:42:37,706 ERROR   wandb-upload_0:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5511', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:37,706 ERROR   wandb-upload_0:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2nqbyuy9/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2nqbyuy9/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E968FA880DD39B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:37,731 ERROR   wandb-upload_3:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/502/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094236Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b9c1ace382d22e8d8135f96a5286dc39071d14a1e388cfe552a72f9717892003: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/502/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094236Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b9c1ace382d22e8d8135f96a5286dc39071d14a1e388cfe552a72f9717892003
2024-08-07 17:42:37,731 ERROR   wandb-upload_3:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:37,732 ERROR   wandb-upload_3:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/502/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/502/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E968FA89C5FD17</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:37,918 ERROR   wandb-upload_1:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93b56afe6d26f7f3606969dc0019d62ba35af93e2a064b0e0bcd513d68456cba: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=93b56afe6d26f7f3606969dc0019d62ba35af93e2a064b0e0bcd513d68456cba
2024-08-07 17:42:37,918 ERROR   wandb-upload_1:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:37,918 ERROR   wandb-upload_1:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2nqbyuy9/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2nqbyuy9/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E968FA949D1398</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:38,110 ERROR   wandb-upload_4:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/502/3ec9d0b0f39e55d0938d525d881bb305?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094236Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b6781eb18bf71ed70b954bedcaf941405d1d7f35bbf9ab8a829dfe61c0f3c284: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/502/3ec9d0b0f39e55d0938d525d881bb305?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094236Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b6781eb18bf71ed70b954bedcaf941405d1d7f35bbf9ab8a829dfe61c0f3c284
2024-08-07 17:42:38,110 ERROR   wandb-upload_4:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'PsnQsPOeVdCTjVJdiBuzBQ==', 'Content-MD5': 'PsnQsPOeVdCTjVJdiBuzBQ==', 'Content-Type': 'application/json', 'Content-Length': '31083', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:38,110 ERROR   wandb-upload_4:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/502/3ec9d0b0f39e55d0938d525d881bb305</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/502/3ec9d0b0f39e55d0938d525d881bb305</Resource><Region>wandb-local</Region><RequestId>17E968FA9877D948</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:38,888 ERROR   wandb-upload_2:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0857eef7e7867127aa5e226e226b37bcd5b929b263db0cf897db75c0c88f5b4f: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/2nqbyuy9/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094233Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=0857eef7e7867127aa5e226e226b37bcd5b929b263db0cf897db75c0c88f5b4f
2024-08-07 17:42:38,888 ERROR   wandb-upload_2:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '284630', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:38,888 ERROR   wandb-upload_2:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/2nqbyuy9/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/2nqbyuy9/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E968FAC9C79A7D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:42:38,955 ERROR   wandb-upload_3:5697 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/502/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094236Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b9c1ace382d22e8d8135f96a5286dc39071d14a1e388cfe552a72f9717892003: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/502/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T094236Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=b9c1ace382d22e8d8135f96a5286dc39071d14a1e388cfe552a72f9717892003
2024-08-07 17:42:38,955 ERROR   wandb-upload_3:5697 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:42:38,956 ERROR   wandb-upload_3:5697 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/502/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/502/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E968FAD2C2801F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
